packages/message/src/Granules.ts

```diff
@@ -23,3 +24,19 @@ import { Message } from '@cumulus/types';
 export const getMessageGranules = (
   message: Message.CumulusMessage
 ): unknown[] | undefined => get(message, 'payload.granules');
+
+/**
+ * Get selective cnm fields from execution message
+ *
+ * @param {Message.CumulusMessage} message - An execution message
+ * @returns {Object|undefined} object with cnm message fields
+ *
+ * @alias module:Granules
+ */
+export const getMessageCnm = (
+  message: Message.CumulusMessage
+): object | undefined => {
+  const fields = ['identifier', 'submissionTime', 'receivedTime', 'processCompleteTime'];
```

Comment:
It will take more space to save entire `cnm` object.  Here is an example of cnmResponse:
    "cnmResponse": {
      "version": "1.5",
      "provider": "PODAAC_SWOT_test-jl-tf-KinesisTestTrigger-1608222016169",
      "collection": "L2_HR_PIXC_test-jl-tf-KinesisTestTrigger-1608222016169",
      "submissionTime": "2017-09-30T03:42:29.791198",
      "identifier": "87b06d565107f6a00c2fbe2530f6d05d49c2527a",
      "product": {
        "dataVersion": "000",
        "files": [
          {
            "type": "data",
            "name": "L2_HR_PIXC_product_0001-of-4154.h5",
            "uri": "https://56d2batozk.execute-api.us-east-1.amazonaws.com/DEV/jl-test-integration-protected/L2_HR_PIXC___000/L2_/L2_HR_PIXC_product_0001-of-4154.h5",
            "checksumType": "md5",
            "checksum": "cd043418380acdf69b2fb80649357cda",
            "size": 50
          },
          {
            "type": "metadata",
            "name": "L2_HR_PIXC_product_0001-of-4154_test-jl-tf-KinesisTestTrigger-1608222016169.cmr.xml",
            "uri": "https://56d2batozk.execute-api.us-east-1.amazonaws.com/DEV/jl-test-integration-protected/L2_HR_PIXC___000/L2_/L2_HR_PIXC_product_0001-of-4154_test-jl-tf-KinesisTestTrigger-1608222016169.cmr.xml",
            "size": 2391
          }
        ],
        "name": "L2_HR_PIXC_product_0001-of-4154_test-jl-tf-KinesisTestTrigger-1608222016169"
      },
      "receivedTime": "2020-12-17T16:22:10.826Z",
      "response": {
        "status": "SUCCESS",
        "ingestionMetadata": {
          "catalogId": "G1239428954-CUMULUS",
          "catalogUrl": "https://cmr.uat.earthdata.nasa.gov/search/granules.json?concept_id=G1239428954-CUMULUS"
        }
      },
      "processCompleteTime": "2020-12-17T16:23:07.744Z"
    }
  }